Xamarin.Forms.Core/TemplatedItemsList.cs

```diff
@@ -188,7 +188,7 @@ internal ListViewCachingStrategy CachingStrategy
 			}
 		}
 
-		internal IListProxy ListProxy
+		public IListProxy ListProxy
```

Comment:
Not very explicit what ListProxy is.. but i understand the change, and i don't know other name. Users could now mess with our proxy enumerable